src/lib/controller/incoming_message.ts

```diff
@@ -31,12 +43,20 @@ export interface IncomingCommandControllerRemoveFailedNode
   nodeId: number;
 }
 
-export interface IncomingCommandControllerReplaceFailedNode
-  extends IncomingCommandControllerBase {
-  command: ControllerCommand.replaceFailedNode;
-  nodeId: number;
-  includeNonSecure?: boolean;
-}
+export type IncomingCommandControllerReplaceFailedNode =
```

Comment:
Why is this command two commands ?

Reply:
It's to add backwards compatibility. The command name is the same, but the old signature and the new signature don't match

Reply:
I think that it's better to define 2 interfaces to make that clear. That way it's also easy to cast it to that interface in our controller.

Reply:
I tried doing the casting in the processInclusionOptions function but I wasn't able to make it work